### PR TITLE
Try to fix Opensuse certificate errors.

### PR DESCRIPTION
**Pull request by fruffy-bfn**
The OpenSuse repository keeps failing to download the p4lang debian packages. Maybe updating the deprecated key installation technique can help?

@rst0git I can not quite figure out how to fix this. Do maybe know how to fix the key installation issue? 

I believe the OpenSuse repository is malformed? It does not have a `dists` folder. I managed to work around that, but I still run into key issues.

### CODE REVIEW

**Comment by rst0git** [comment]
There are updated installation instructions at
https://software.opensuse.org//download.html?project=home%3Ap4lang&package=p4lang-p4c
```
echo 'deb http://download.opensuse.org/repositories/home:/p4lang/xUbuntu_${DISTRIB_RELEASE}/ /' | sudo tee /etc/apt/sources.list.d/home:p4lang.list
curl -fsSL https://download.opensuse.org/repositories/home:p4lang/xUbuntu_${DISTRIB_RELEASE}/Release.key | gpg --dearmor | sudo tee /etc/apt/trusted.gpg.d/home_p4lang.gpg > /dev/null
sudo apt-get update
```

**Comment by fruffy-bfn** [comment]
This does seem to work! Thanks for pointing to the right instructions, could not figure out how install the key.

**Comment by rst0git** [comment]
It might be good to also update the [README.md](https://github.com/p4lang/p4c/blob/bf6efdd22e21b6cfeac358e23348ab133327eb50/README.md?plain=1#L115) file.

File: README.md [hunk 1]
```diff
@@ -111,8 +111,8 @@ A p4c package is available in the following repositories for Ubuntu 20.04 and ne
 
 ```bash
 . /etc/os-release
```
Comment (fruffy) [inline README.md:113]: This should probably be lsb_release, will fix this later.

**Review by fruffy — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (rst0git) [reply README.md]: yes
```
$ ls -al /etc/os-release 
lrwxrwxrwx. 1 root root 21 Mar 14 11:20 /etc/os-release -> ../usr/lib/os-release
```

**Pull request merged**